index.html

```diff
@@ -27,22 +33,204 @@ <h1>I'm Ndayishimiye Glad to see you</h1>
+          <img src="./img/work4.svg" alt="Project four" />
+        </div>
+        <div class="item-title">Multi-Post Stories</div>
+        <div class="item-history">
+          <span class="t-bold">Canopy</span>
+          <img class="counter" src="./img/Counter.svg" alt="counter" />
+          <span>Back End Dev</span>
+          <img class="counter" src="./img/Counter.svg" alt="counter" />
+          <span>2022</span>
+        </div>
+        <div class="item-desc">
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+        </div>
+        <ul class="item-categories">
+          <li class="item-category">html</li>
+          <li class="item-category">css</li>
+          <li class="item-category">javascript</li>
+        </ul>
+        <button class="btn-primary">See Project</button>
+      </div>
+    </section>
+    <section class="about container">
+      <div class="about-me">
+        <h1>
+          About <br />
+          Myself
+        </h1>
+        <p>
+          Hello I'm a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don't hestiate to
+          contact me.
+        </p>
+        <span class="desc-span">LET'S CONNECT</span>
+        <ul class="connect-icons">
+          <li class="icon">
+            <a href="https://twitter.com/odaltongain"
+              ><img src="./img/twitter.svg" alt="Twitter"
+            /></a>
+          </li>
+          <li class="icon">
+            <a href="https://linkedin.com/in/nderic"
+              ><img src="./img/Linkedinicon.svg" alt="Linkedin"
+            /></a>
+          </li>
+          <li class="icon">
+            <a href="#"><img src="./img/medium.svg" alt="Medium" /></a>
+          </li>
+          <li class="icon">
+            <a href="https://github.com/ndayishimiyeeric"
+              ><img src="./img/github.svg" alt="github"
+            /></a>
+          </li>
+          <li class="icon">
+            <a href="#"><img src="./img/peace.svg" alt="social" /></a>
+          </li>
+        </ul>
+        <button class="btn-primary">Get my resume</button>
+      </div>
+      <ul class="web-dev">
+        <li class="language">
+          <div class="language-title">
+            Languages
+            <img src="./img/downarrow.svg" alt="downward arrow" />
+          </div>
+          <div class="language-cards">
+            <div class="language-card">
+              <img src="./img/js.svg" alt="javascript" />
+              <p>javascript</p>
+            </div>
+            <div class="language-card">
+              <img src="./img/html.svg" alt="html" />
+              <p>HTML</p>
+            </div>
+            <div class="language-card">
+              <img src="./img/css.svg" alt="css" />
+              <p>CSS</p>
```

Comment:
- [ ] I think It would be better to add the Technology in each category of skills a list using the `<ul>` tag and each of them in a` li` tag. Kindly, fix them.